01-node-tutorial/answers/practice2.js

```diff
@@ -0,0 +1,3 @@
+const singleVariable = '\nThis is my sentence from practice2.js \n'
+
+module.exports = {singleVariable}
```

Comment:
Challenge question: How would you need to change `practice2.js` if you removed the curly braces in the export:

```javascript
module.exports = singleVariable
```

Reply:
The simplest change here would be 
`module.exports.singleVariable = singleVariable`
It would be the same as using the line of code below
`exports.singleVariable = singleVariable`

Reply:
Ok good idea.

On the caller side you could also change 

```
const { singleVariable }= require('./practice2')
```

To just 

```
const singleVariable = require('./practice2')
```